.rusty-hook.toml

```diff
@@ -1,5 +1,5 @@
 [hooks]
-pre-push = "cargo +nightly fmt --all -- --check"
+pre-push = "rustup run nightly cargo fmt --all -- --check"
```

Comment:
Seems, that this change is unrelated to the PR

Reply:
I couldn't push my branch until this was changed. I'm not sure unless there's a good overall solution for rust committers, it should be done this way.

Reply:
Is `cargo` in your environment?